Python/tests/test_RerF.py

```diff
@@ -164,3 +164,69 @@ def test_fastPredictPost_X_Y_input():
     # pred class should equal the max prob in posteriors
     assert np.array_equal(pred_all.argmax(axis=1), fastPredict(feat_data, forest))
 
+
+
+def test_S_RerF():
+
+    data_fname = "packedForest/res/mnist.csv"  # mnist
+    train_X = np.genfromtxt(data_fname, delimiter=",")
+    label_col = 0
+
+    feat_data = train_X[:, 1:]  # mnist
+    labels = train_X[:, label_col]
+
+    ## Subset the data to for time.
```

Comment:
i think i know what you mean here, but specifically, it does not make sense

Reply:
fixed, thanks.